src/evm.rs

```diff
@@ -43,11 +43,20 @@ pub struct CallRawResult {
 
 impl From<CallTraceNode> for CallTrace {
     fn from(item: CallTraceNode) -> Self {
+        let data = match item.trace.kind {
+            CallKind::Call | CallKind::StaticCall => {
+                let first_4_bytes: Vec<u8> =
+                    item.trace.data.to_raw().iter().take(4).cloned().collect();
+                Bytes::from_iter(first_4_bytes)
+            }
+            _ => Bytes::from_iter(vec![0]),
```

Comment:
vec![0] is macro to create a Bytes object with 0 mean single zero byte then we should use `Bytes::from` instead of  `Bytes::from_iter`